policy/modules/system/modutils.te

```diff
@@ -40,6 +40,8 @@ dontaudit kmod_t self:capability sys_admin;
 
 allow kmod_t self:udp_socket create_socket_perms;
 allow kmod_t self:rawip_socket create_socket_perms;
+allow kmod_t self:lockdown confidentiality;
```

Comment:
I'd prefer not to add anymore `lockdown` rules, since the checks will be removed in newer kernels, so it should be removed.

Optionally, remove all lockdown rules from the policy and then add `allow domain self:lockdown { confidentiality integrity };` in `domain.te`.

Reply:
Thanks. It's OK for me. How about the 2nd patch ?

Reply:
The key write is ok.